### PR TITLE
add user-admin-server

### PR DESCRIPTION
**Pull request by etiennehomer**
Signed-off-by: Etienne Homer <etiennehomer@gmail.com>

### CODE REVIEW

File: k8s/resources/common/user-admin-server-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+kind: Deployment
+metadata:
+  name: user-admin-server
+  labels:
+    name: user-admin-server
+    version: "1"
+    app.kubernetes.io/component: gridsuite-springboot
+    gridsuite.org/springboot-with-database: "true"
+  annotations:
+    gridsuite.org/size: springboot-l
```
Comment (jonenst) [inline k8s/resources/common/user-admin-server-deployment.yaml:11]: xs

File: k8s/resources/study/config/study-server-application.yml [hunk 2]
```diff
@@ -2,3 +2,7 @@ spring:
   data:
     elasticsearch:
       enabled : true
+      
+backing-services:
+  single-line-diagram:
+    base-uri: http://172.17.0.1:5005/
```
Comment (jonenst) [inline k8s/resources/study/config/study-server-application.yml:8]: remove ?

File: k8s/resources/common/user-admin-server-deployment.yaml (hunk 1 above)
Comment (etiennehomer) [reply k8s/resources/common/user-admin-server-deployment.yaml]: done

File: k8s/resources/study/config/study-server-application.yml (hunk 2 above)
Comment (etiennehomer) [reply k8s/resources/study/config/study-server-application.yml]: done

**Review by jonenst — APPROVED** [review]

**Pull request merged**